docs/root/configuration/cluster_manager/cluster_runtime.rst

```diff
@@ -10,13 +10,13 @@ Active health checking
 
 health_check.min_interval
   Min value for the health checking :ref:`interval <envoy_api_field_core.HealthCheck.interval>`.
-  Default value is 0. The health checking interval will be between *min_interval* and
-  *max_interval*.
+  Default value is 1 ms. Values less than 1 ms will be treated as 1 ms. The health checking interval
```

Comment:
This seems like a complicated way of saying that the minimum value of the field is 1 ms, but I think what you really mean is after applying jitter values less than 1 ms will be treated as 1 ms. Is there a better way of saying this here?